yarn-project/noir-protocol-circuits-types/src/conversion/server.ts

```diff
@@ -469,22 +469,6 @@ export function mapPrivateToPublicAccumulatedDataToNoir(
   };
 }
 
-export function mapPrivateToRollupAccumulatedDataToNoir(
-  privateToRollupAccumulatedData: PrivateToRollupAccumulatedData,
-): PrivateToRollupAccumulatedDataNoir {
-  return {
-    note_hashes: mapTuple(privateToRollupAccumulatedData.noteHashes, mapFieldToNoir),
-    nullifiers: mapTuple(privateToRollupAccumulatedData.nullifiers, mapFieldToNoir),
-    l2_to_l1_msgs: mapTuple(privateToRollupAccumulatedData.l2ToL1Msgs, mapScopedL2ToL1MessageToNoir),
-    private_logs: mapTuple(privateToRollupAccumulatedData.privateLogs, mapPrivateLogToNoir),
-    contract_class_logs_hashes: mapTuple(
-      privateToRollupAccumulatedData.contractClassLogsHashes,
-      mapScopedLogHashToNoir,
-    ),
-    contract_class_log_preimages_length: mapFieldToNoir(privateToRollupAccumulatedData.contractClassLogPreimagesLength),
-  };
-}
-
```

Comment:
I moved this to `common` - I didn't think it made sense to have `mapPrivateToRollupAccumulatedDataToNoir` and `mapPrivateToRollupAccumulatedDataFromNoir` in different files